.github/workflows/twister_dfu_nrf52840dk.yml

```diff
@@ -124,6 +124,7 @@ jobs:
           source env_dfu_nrf52840dk.sh
           PATH=$PATH:/opt/nrf-command-line-tools/bin
           nrfjprog -i
+          golioth releases rollback '*'
```

Comment:
I thought about that, and two things drove this decision:

* removing releases is destructive (you can't get the deleted binaries back),  but rollback can be undone.
* I think if v2.0.0 exists, we probably want the test to fail because it indicates some previous trouble that we should pay attention to

If you feel strongly about deleting instead of rolling back, let me know and I'll change it.

Reply:
There should be no other than 2.0.0 releases/artifacts, as pytest should be the only user of this project/account. Right? So even when `golioth releases rollback '*'` will disable DFU of the device, the test will still fail due to existing artifacts/releases. So the only difference it makes is that the test will fail in a different way, but it **will** fail. This is why I would either do artifacts/releases removal **or** nothing at all. Does it make sense?